README.md

```diff
@@ -33,36 +33,96 @@ yarn add react-dropzone
 
 ## Usage
 
-Import `Dropzone` in your React component:
-
 ```javascript static
+import React from 'react'
+import classNames from 'classnames'
 import Dropzone from 'react-dropzone'
+
+class MyDropzone extends React.Component {
+   onDrop = (acceptedFiles, rejectedFiles) => {
+     // Do something with files
+   }
+
+   render() {
+    return (
+      <Dropzone onDrop={this.onDrop}>
+        {({getRootProps, getInputProps, isDragActive}) => {
+          return (
+            <div
+              {...getRootProps()}
+              className={classNames('dropzone', {'dropzone--isActive': isDragActive})}
+            >
+              <input {...getInputProps()} />
+              {
+                isDragActive ?
+                  <p>Drop files here...</p> :
+                  <p>Try dropping some files here, or click to select files to upload.</p>
+              }
+            </div>
+          )
+        }}
+      </Dropzone>
+    );
+  }
+}
 ```
 
-  and specify the `onDrop` method that accepts two arguments. The first argument represents the accepted files and the second argument the rejected files.
+## Render Prop Function
 
-```javascript static
-function onDrop(acceptedFiles, rejectedFiles) {
-  // do stuff with files...
-}
+The render property function is what you use to render whatever you want to based on the state of `Dropzone`.
+
+You can pass it as a child:
+```jsx static
+<Dropzone>
+  {({getRootProps}) => <div {...getRootProps()} />}
+</Dropzone>
+```
+
+Or as a prop of the dropzone:
+```jsx static
+<Dropzone
+  render={({getRootProps}) => <div {...getRootProps()} />}
```

Comment:
Why support `render` even?

Reply:
Good point, it was in the prev PR, didn't think to remove it.